.github/workflows/cmd-run.yml

```diff
@@ -242,13 +241,30 @@ jobs:
             --ignore-errors \
             refs/remotes/origin/master $PR_BRANCH)
 
+          echo $result
+
+          echo $result > /tmp/cmd/subweight.log
+          # Though github claims that it supports 1048576 bytes in GITHUB_OUTPUT in fact it only supports ~200000 bytes of a multiline string
```

Comment:
Can we avoid using `GITHUB_OUTPUT` here at all?

What about generating/storing `subweight` output to the file, which we upload `actions/upload-artifact@v4` and then later, when needed, we do just download `actions/download-artifact@v4` or access/read that stored file - e.g. in the step `name: Comment PR (End)`

Reply:
Yes, we can, but I thought that it's convenient to read it in a comment, when there are no so many lines. And I already put it into the file and upload it as an artifact when there are too many lines. The problem now is that is writes everything into one line :/